app/src/main/java/chat/rocket/android/model/ServerConfig.java

```diff
@@ -31,6 +32,7 @@
   private int state;
   private String session;
   private String error;
+  private boolean syncPushToken;
```

Comment:
I was trying to leave server related config vars on the `ServerConfig` class.

Maybe I could remove it somewhere else and rename this class to `ServerConnectivityConfig`??